db/nft.go

```diff
@@ -132,9 +133,9 @@ func GetClassesRanking(conn *pgxpool.Conn, q QueryRankingRequest, p PageRequest)
 	JOIN nft_class AS c
 		ON c.id = t.class_pid
 	GROUP BY c.id
-	ORDER BY total_sold_value DESC
+	ORDER BY %[1]s DESC
 	LIMIT $1
-	`
+	`, orderBy)
```

Comment:
Not sure if we want this to be a param or formatted string @nnkken

Reply:
Using SQL param is not that straight forward, need something like this:
```SQL
ORDER BY
  CASE WHEN $1 = 'sold_count' THEN sold_count END,
  CASE WHEN $1 = 'total_sold_value' THEN total_sold_value END
```
It may also affects the parser and optimizer, so I think format string is fine

Reply:
Thank @nnkken for the explanation. I did encounter this issue when dev, so I use a formatted string here.